packages/theming/atlas/src/theme/native/custom-variables.ts

```diff
@@ -1,3 +1,4 @@
+import { VariablesColumnChart } from "./../../themesource/atlas_core/native/types/variables.d";
```

Comment:
This import is defined twice. You can move the `VariablesColumnChart` to the below import.

Reply:
done